ironfish-cli/src/commands/accounts/export.ts

```diff
@@ -47,8 +50,34 @@ export class ExportCommand extends IronfishCommand {
 
     if (exportPath) {
       const resolved = this.sdk.fileSystem.resolve(exportPath)
-      fs.writeFileSync(resolved, output)
-      this.log(`Exported account ${account} to the file ${exportPath}`)
+
+      try {
+        const stats = await fs.promises.stat(resolved)
+        if (stats.isDirectory()) {
+          await fs.promises.writeFile(
+            this.sdk.fileSystem.join(resolved, `ironfish-${account}.txt`),
+            output,
+          )
+        } else {
+          this.log(`There is already an account backup at ${exportPath}`)
+          const confirmed = await cli.confirm(
+            `\nOverwrite the account backup with new file?\nAre you sure? (Y)es / (N)o`,
+          )
+          if (!confirmed) {
+            this.exit(1)
```

Comment:
Just a side note, but you don't need to put the write file in an else statement because this.exit() returns never, because it calls process.exit()

Reply:
Oh yeah, good catch for sure.